pkg/tests/ossm/ior_test.go

```diff
@@ -0,0 +1,322 @@
+package ossm
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/env"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	"github.com/maistra/maistra-test-tool/pkg/util/version"
+)
+
+type RouteMetadata struct {
+	Uid             string            `json:"uid"`
+	Name            string            `json:"name"`
+	ResourceVersion string            `json:"resourceVersion"`
+	Labels          map[string]string `json:"labels"`
+	Annotations     map[string]string `json:"annotations"`
+}
+
+type Route struct {
+	Metadata RouteMetadata `json:"metadata"`
+}
+
+const (
+	originalHostAnnotation = "maistra.io/original-host"
+)
+
+// TestIOR tests IOR error regarding routes recreated: https://issues.redhat.com/browse/OSSM-1974. IOR will be deprecated on 2.4 and willl be removed on 3.0
```

Comment:
Why not move this comment into `t.Log()` so it's visible when you run the test?

Reply:
Done